Umbraco-Cloud/Set-Up/Dedicated/index.md

```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+versionTo: 10.0.0
+---
+
+# Move your Umbraco Cloud project to dedicated resources
+
+In this article, you can read about how you enable your Umbraco Cloud project to run on dedicated resources and what you need to be aware of before you do so.
+
+## Before you move your project to dedicated resources
+
+Before you decide to move your Umbraco Cloud project, you need to consider a few things:
+
+- Currently, Umbraco Cloud offers dedicated resources for professional plans only. We expect to include dedicated resources for Starter and Standard plans in the self-service feature in Umbraco Cloud portal later in 2022.
+
+- In case you want to move your Umbraco Cloud project with a Starter or Standard project plan to a dedicated server in Umbraco Cloud, please reach out to our support who will assist you in doing this.
+
+- Moving from a shared plan to a dedicated plan will change the outgoing IP of the project. If your solution has an external service that requires whitelisting the outgoing IP of the project, we advise you to enable the static outbound IP feature for your cloud project and share that static outbound IP address with the third party. The static outbound IP address will not change when moving from a shared plan to a dedicated resource. For more info on static lease visit the documentation for [external services](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/External-Services/).
+
+
+## How to upgrade your plan
+
+The first step moving to a dedicated resource is to access your project in the project overview at [Umbraco.io](https://www.s1.umbraco.io/projects).
+
+- In the project overview you can find all the projects that you have been invited to or have created.
+
+- From here you need to find the project that you want to move to dedicated.
+
+Under the project on the right side you have a dropdown menu called settings:
+
+![Upgrade plan step 1](images/Step1.png)
+
+- Clicking on the tab will direct you to the _Dedicated resource_ page that guides you on how to upgrade to a dedicated plan.
+
+- For a Umbraco Cloud project with a Shared Professional plan there are three dedicated plans for you to choose among. For each of the dedicated plans, you will find its name, the memory and CPU cores, and the price per month.
```

Comment:
Does this mean that there are other options for other plans? 🤔 
As the feature is only available for pro plans at the moment, I think we can leave out the part about how this particular plan has these specific options.

We can always specify the options when we expand the feature.

Reply:
I am not sure that I entirely understand the reasoning for not having this info that there are three dedicated plans to choose from.

Reply:
In the latest commit, I changed the term "dedicated plan" to "dedicated option" (and "shared plan" to "shared resource") as this is a better term and won't collide with our normal use of the "plan". That will help quite a bit with the understanding, I believe :-).

We definitely could add a "currently" in there as well, if that will help.